src/app/shared/components/icon-button/icon-button.component.html

```diff
@@ -1 +1 @@
-<fa-icon [title]="tooltipText" [icon]="iconButtonProp" [size]="iconButtonSizeProp"></fa-icon>
+<fa-icon [title]="tooltip" [icon]="iconButtonProp" [size]="iconButtonSizeProp"></fa-icon>
```

Comment:
Why is this renamed?

Reply:
To streamline it?

Reply:
Against we don't needlessly add pre- and postfix on stuff :)